config/default_settings.yaml

```diff
@@ -0,0 +1,17 @@
+# Default arguments used for the CLI. Can be overridden using the corresponding
+# flag at runtime
+default_args:
+  run_github_workflow: True
+  run_glue_crawler: True
+  upload_logs: True
+  upload_data: True
+
+# Data type overrides are necessary to maintain parity/consistency with sqoop.
+# Schema overrides are defined in three places, with each overriding the last:
+#   1. By default, NUMBER Oracle types are converted to DECIMAL(10,0)
+#      and TIMESTAMP Oracle types are converted to STRING
+#   2. Global schema overrides, as defined below
+#   3. Table schema overrides, defined in table_definitions.yaml
```

Comment:
**[Suggestion, non-blocking]** Very nice! At some point it might be nice to document this config system in the top-level README, but I don't feel strongly about that happening right now. We might just want to take a final pass at the docs once this repo is in a stable state.

Reply:
I have a follow-up PR for the README right after this one!